AuthenticatorShared/UI/Vault/ItemList/ItemList/ItemListProcessorTests.swift

```diff
@@ -1104,4 +1104,32 @@ class ItemListProcessorTests: AuthenticatorTestCase { // swiftlint:disable:this
         waitFor(subject.state.itemListCardState == .passwordManagerSync)
         task.cancel()
     }
+
+    /// Tests that the `itemListCardState` is set to `none` if the user has already enabled sync in the PM app.
```

Comment:
🎨 I think it would be helpful to clarify in the comments of these two tests what the difference between them is